src/main/java/pl/debuguj/parkingspacessystem/config/DriverTypeConverter.java

```diff
@@ -0,0 +1,14 @@
+package pl.debuguj.parkingspacessystem.config;
+
+import pl.debuguj.parkingspacessystem.enums.DriverType;
+
+import java.beans.PropertyEditorSupport;
+
+/**
+ * Created by grzesiek on 17.10.17.
+ */
+public class DriverTypeConverter extends PropertyEditorSupport {
```

Comment:
Do czego to jest potrzebne?

Reply:
@pivovarit Wykorzystywałem to do walidacji enuma DriverType z url
@MateuszRasinski Oki, sprawdzę